app/uk/gov/hmrc/helptosavefrontend/models/register/CreateAccountRequest.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosavefrontend.models.register
+
+import play.api.libs.json.{Format, Json}
+import uk.gov.hmrc.helptosavefrontend.models.userinfo.NSIUserInfo
+
+case class CreateAccountRequest(userInfo: NSIUserInfo, eligibilityReason: Option[Int])
```

Comment:
I think in all cases in the digital FE we're gonna have the `eligibilityReason` when creating accounts - this can safely be an `Int` instead of an `Option[Int]`